src/jobrunner.rs

```diff
@@ -109,6 +108,12 @@ impl JobRunner {
             // See if any of our active jobs have events. Knowing when a pipe is actually closed is
             // surprisingly hard. https://www.greenend.org.uk/rjk/tech/poll.html has an interesting
             // suggestion which we adapt slightly here.
+            //
+            // This `for` loop has various unwrap() calsl. If `flags[i * 2]` or `flags[i * 2 + 1]`
```

Comment:
`calsl`

Reply:
Fixed in https://github.com/softdevteam/snare/pull/28/commits/ce102a545852a819d497cc57063a68db3548e347.